crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -317,47 +312,26 @@ impl CumulocityConverter {
                 }
                 _ => {
                     // if the target entity is unregistered, try to register it first using auto-registration
```

Comment:
I guess this comment is no longer applicable.

Reply:
Resolved by d88c054